docs/cloud/features/scheduler/scheduler.md

```diff
@@ -163,8 +163,35 @@ This gives you complete control over data security and network access while stil
 
 ### How it works
 
-Coming soon!
+Self-hosted executors as the name indicates are self-hosted workers that take on the responsibility of "executing" changes to the data warehouse. While Tobiko Cloud schedules and plans changes, the executors are responsible for executing those changes. Executors are docker containers that are configured to connect to Tobiko Cloud as well as your data warehouse. Connected to both, the executor will pull the work, whether it's a plan or scheduled background work from a run, and execute it on your data warehouse.
 
 ### Configuration
 
-Coming soon!
\ No newline at end of file
+Exact configuration is left to the user and will vary based on the infrastructure and setup of the user, for example the executors could be run on a Kubernetes cluster or as a standalone pair of Docker containers depending on the user's infrastructure.
```

Comment:
Can you provide example configs for these pieces?

public ingress, k8s resources, etc.

Reply:
Do you by default recommend a Kubernetes cluster?

What's the minimum spec needed?

Reply:
Notes for data engineer requesting from infrastructure/security team:

- 2 docker containers (tcloud image)
- ability to call datawarehouse
- ability to call tcloud
- specificied entrypoint
- environment variables set
- monitored